### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**WISHLIST MANAGEMENT**
- Wishlist page
- Wishlist context
- Wishlist reducer
- Add items to wishlist
- Remove items from wishlist
- fix products grid default stretch

### CODE REVIEW

File: src/components/cartpage/CartItems.js [hunk 1]
```diff
@@ -7,18 +7,22 @@ const CartItems = () => {
   const { cartState, cartDispatch } = useCart();
   console.log(cartState);
```
Comment (tushartiwari7) [inline src/components/cartpage/CartItems.js:8]: suggestion: remove console logs before pushing to production.

File: src/components/cartpage/CartItems.js [hunk 2]
```diff
@@ -7,18 +7,22 @@ const CartItems = () => {
...
       {cartState.map((product) => (
-        <div className="cart-item">
-          <div class="cover-img">
-            <img class="cart-item-img" src={product.image} alt="first tshirt" />
+        <div className="cart-item" key={product._id}>
+          <div className="cover-img">
+            <img
+              className="cart-item-img"
+              src={product.image}
+              alt="first tshirt"
```
Comment (tushartiwari7) [inline src/components/cartpage/CartItems.js:17]: make the alt tag dynamic as currently all the products will have same ali text.

**Review by tewarimohit — APPROVED** [review]
Good to merge...

File: src/components/productListing/Header.js [hunk 3]
```diff
@@ -1,4 +1,3 @@
-// import "./Header.css";
 import searchIcon from "../../assets/nav-icons/search.svg";
 import likeIcon from "../../assets/nav-icons/heart.svg";
 import loginIcon from "../../assets/nav-icons/user-plus.svg";
```
Comment (tushartiwari7) [inline src/components/productListing/Header.js:3]: try exporting all icons from a single file if you are not using any external library for icons.

**Review by tushartiwari7 — APPROVED** [review]
Added some suggestions.

File: src/components/cartpage/CartItems.js (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/cartpage/CartItems.js:8]: yes , I will remove them all

File: src/components/cartpage/CartItems.js (hunk 2 above)
Comment (tanishk15sharma) [reply src/components/cartpage/CartItems.js:17]: okay sure , thanks for this issue

File: src/components/productListing/Header.js (hunk 3 above)
Comment (tanishk15sharma) [reply src/components/productListing/Header.js:3]: okay sure I will do it

**Pull request merged**